src/main/resources/templates/index.twig

```diff
@@ -11,4 +11,11 @@
     </li>
     {% endfor %}
 </ul>
-{% endblock %}
\ No newline at end of file
+<form method="POST">
```

Comment:
add `action=polls/create` or something, and IMO it should be in that controller, not in the home controller